e2e/e2e_test.go

```diff
@@ -0,0 +1,105 @@
+package e2e
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+	"os"
+	"os/exec"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+const MtlsPK = "mtls.key"
+const MtlsCERT = "mtls.crt"
+
+var _ = Describe("Mock test", func() {
+
+	var mtlscertFile, mtlskeyFile *os.File
+	var err error
+	var tmpdir, compiledStupid string
+	var stupidSession *gexec.Session
+
+	BeforeSuite(func() {
+
+		tmpdir, err = ioutil.TempDir("", "stupid-e2e-")
+
+		mtlscertFile, err = ioutil.TempFile(tmpdir, MtlsCERT)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		mtlskeyFile, err = ioutil.TempFile(tmpdir, MtlsPK)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		err = GenerateCertAndKeys(mtlskeyFile, mtlscertFile)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		mtlscertFile.Close()
+		mtlskeyFile.Close()
+
+		compiledStupid, err = gexec.Build("../cmd/stupid")
+		Expect(err).ShouldNot(HaveOccurred())
+	})
+
+	Context("end to end test with mocked Fabric", func() {
+
+		It("should running successfully when no tls", func() {
+			config, err := ioutil.TempFile("", "configmock.yaml")
+			configValue := Values{Priv_sk: mtlskeyFile.Name(), Sign_cert: mtlscertFile.Name(), Mtls: false}
+			GenerateConfigFileForTest(config.Name(), configValue)
+
+			grpcServer := grpc.NewServer()
+			defer grpcServer.Stop()
+
+			go StartMock(grpcServer)
+			stupid_command := exec.Command(compiledStupid, config.Name(), "500")
```

Comment:
camelCase instead of snake_case

and actually this var name could just be `cmd`